homeassistant/components/myfitnesspal/sensor.py

```diff
@@ -0,0 +1,68 @@
+"""Support for Myfitnesspal totals as sensors."""
+from datetime import date
+from homeassistant.const import MASS_GRAMS
+from homeassistant.helpers.entity import Entity
+
+# REQUIREMENTS = ['myfitnesspal==1.13.3']
+TOTALS = ["sodium", "carbohydrates", "calories", "fat", "sugar", "protein"]
+
+
```

Comment:
do we need a config schema?

Reply:
I can certainly add one. To choose which of the set to track?

Reply:
I think the bast config schema would be username and password

Reply:
Believe this is covered now.

Reply:
Believe this is covered now.